_test/test_algorithms/test_cut.m

```diff
@@ -58,25 +59,6 @@
             end
         end
         %
-        function test_take_a_cut_from_an_sqw_file_single_chunk(obj)
-            % Really large file V2 on disk to ensure that ranges are
-            % calculated using filebased algorithm rather than all data
-            % loaded in memory.
-            %v2large_file= 'c:\Users\abuts\Documents\Data\Fe\Data\sqw\Fe_ei1371_base_a.sqw';
-            %sqw_cut = cut(v2large_file, obj.ref_params{:});
-
-            conf = hor_config();
-            old_conf = conf.get_data_to_store();
-            conf.mem_chunk_size = 8000;
-            cleanup = onCleanup(@() set(hor_config, old_conf));
-
-            sqw_cut = cut(obj.sqw_file, obj.ref_params{:});
-
-            ref_sqw = read_sqw(obj.ref_cut_file);
-            assertEqualToTol(sqw_cut, ref_sqw, 1e-5, ...
-                'ignore_str', true,'-ignore_date');
-        end
```

Comment:
why this have disappeared? It is important test for essential functionality

Reply:
Sorry, it got shoved down to the end of the file while fixing a conflict. Now back in original position. Thanks